src/Domain.LinnApps/PurchaseOrders/PurchaseOrderDetail.cs

```diff
@@ -29,5 +31,46 @@ public class PurchaseOrderDetail
         public string StockPoolCode { get; set; }
 
         public PurchaseOrder PurchaseOrder { get; set; }
+
+        public int? OriginalOrderNumber { get; set; }
+
+        public int? OriginalOrderLine { get; set; }
+
+        public string OurUnitOfMeasure { get; set; }
+
+        public string OrderUnitOfMeasure { get; set; }
+
+        public decimal? Duty { get; set; }
+
+        public decimal? OurUnitPriceCurrency { get; set; } //our price
+        //from mini order trigger:
+        //next_our_unit_price = :new.our_price,
+        //next_order_unit_price = :new.order_price,
+        public decimal? OrderUnitPriceCurrency { get; set; } //order  price
+
+        public decimal? BaseOurUnitPrice { get; set; }
+
+        public decimal? BaseOrderUnitPrice { get; set; }
+
+        public decimal? VatTotalCurrency { get; set; }
+
+        public decimal? BaseVatTotal { get; set; }
+
+        public decimal? DetailTotalCurrency { get; set; }
+
+        public decimal? BaseDetailTotal { get; set; }
+
+        public string DeliveryInstructions { get; set; }
+
+        public Employee DeliveryConfirmedBy { get; set; }
+
+        public int DeliveryConfirmedById { get; set; }
+
+        //todo map and uncomment these
+        //public CancelledPODetail CancelledDetail { get; set; }
```

Comment:
might as well just do this todo if its a two second job

Reply:
Yeah, brain was fried and I wanted to nip outside, but wanted to get this up before everyone finished for the day so I left some bits. Turned out there were actually less todos left than I thought